swift/Tests/Blockchains/TronTests.swift

```diff
@@ -10,10 +10,7 @@ import TrustWalletCore
 class TronTests: XCTestCase {
 
     func testTronAddress() {
-        let blockchain = Tron()
-        let address = blockchain.address(string: "TLWEciM1CjP5fJqM2r9wymAidkkYtTU5k3")
-
-        XCTAssertEqual(address?.description, "TLWEciM1CjP5fJqM2r9wymAidkkYtTU5k3")
-        XCTAssertTrue(address is TronAddress)
```

Comment:
probably don't even need this test anymore

Reply:
iOS still needs it, we should export TWTronAddress later